main.go

```diff
@@ -200,7 +200,7 @@ func calc(stack *stackType, cmd string) error {
 
 		if autoprint {
 			if single {
-				fmt.Println(stack.top()) // plain print to stdout
+				fmt.Println(formatNumber(ctx, stack.top(), ops.base, ops.decimals, true)) // plain print to stdout
```

Comment:
Nit: Avoid inline comments (hard to read, scroll all the way to the right, and no capitalization in this case.)